src/pages/LoginPage.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import LoginTitle from "../assets/images/loginTitle.svg";
+import Background from "../assets/images/loginPageBackground.png";
+
+const styles = `
+    .login-wrapper {
+    background-size: 100%;
+    background-repeat: repeat-y;
+    }
+
+    .login-title{
+        text-align: center;
+        margin-bottom: 40px;
+    }
+
+    .login-input{
+        width: 70%;
+        height: 30px;
+        background-color: #4E9BB9;
+        color: #FFFFFF;
+        font-size: 90%;
+        border-radius: 50px;
+        border: 0px;
+    }
+
+    ::placeholder {
+        color: #FFFFFF;
+    }
+
+    .login-button {
+        width: 70%;
+        height: 30px;
+        background-color: #AAD390;
+        color: #FFFFFF;
+        border-radius: 50px;
+        border: 0px;
+        margin-top: 40px;
+
+    }
+
+    .login-logo {
+        margin-top: 80px;
+        margin-bottom: 25px ;
+    }
+
+    a{
+        color: #91CAD6;
+    }
+`;
+
+function LoginPage(props) {
+    return (
+        <div>
+            <div
+                className="login-wrapper d-flex flex-column"
+                style={{ width: "100%", height: "100vh", backgroundSize: "cover", backgroundImage: `url(${Background})` }}
```

Comment:
```suggestion
                className="login-wrapper d-flex flex-column vh-100 w-100"
                style={{ backgroundSize: "cover", backgroundImage: `url(${Background})` }}
```

- O Bootstrap tem classes pra essas propriedades, pode ajudar no futuro 😊